app/src/main/java/com/vci/vectorcamapp/core/domain/collector/use_cases/CollectorValidationUseCases.kt

```diff
@@ -1,8 +1,8 @@
-package com.vci.vectorcamapp.settings.domain.use_cases
+package com.vci.vectorcamapp.core.domain.collector.use_cases
```

Comment:
The file hierarchy should be use_cases.collector, not the other way around. Swap the folders.